app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -137,17 +135,16 @@ def get_meetings_disabled
   end
 
   def create
-    params.require(:meetingID)
-
+    meeting_id = require_meeting_id
```

Comment:
Creating the new function `require_meeting_id` is redundant, since it just runs `params.require(:meetingID)` the same as the code it's replacing!

If you really want `meeting_id` in a separate variable, then you could have done
```ruby
meeting_id = params.require(:meetingID)
```

That said, this change is also unrelated to the topic of the PR.

Reply:
it seemed better and more readable to move params.require(:meetingID) to a different method and just calling meeting_id wherever required instead of params[:meeting_id]